lib/CMakeLists.txt

```diff
@@ -520,6 +521,8 @@ add_test(NAME ecl_nnc_export5 COMMAND ecl_nnc_export ${_eclpath}/DualPoro/DUALPO
 add_test(NAME ecl_nnc_export6 COMMAND ecl_nnc_export ${_eclpath}/nestedLGRcase/TESTCASE_NESTEDLGR TRUE)
 add_test(NAME ecl_nnc_export7 COMMAND ecl_nnc_export ${_eclpath}/TYRIHANS/BASE20150218_MULTFLT FALSE)
 
+add_test(NAME ecl_nnc_export_intersect COMMAND ecl_nnc_export_intersect)
```

Comment:
The test is add as part of the loop further up in the file - the `add_test()` commands listed here are for the tests which require (unique) commandline arguments.